lib/model/store.dart

```diff
@@ -465,6 +465,10 @@ class PerAccountStore extends ChangeNotifier with ChannelStore, MessageStore {
         if (user == null) {
           return; // TODO log
         }
+        // Fields on [User] not updated here:
+        //   userId, dateJoined, isBot, botType, isSystemBot
```

Comment:
Do we want to have a reference on `User`, so that we remember to update this comment when there is another immutable field?

Reply:
Well, the more important case is when adding a mutable field, so that we add the code to update it.

Anyway, switched to a comment on `User`, in conjunction with making the immutable fields `final`.